### PR TITLE
feat: add subgraph versioning v2

### PR DESCRIPTION
**Pull request by AlcibiadesCleinias**
bump readme: https://github.com/fluencelabs/deal/blob/5ef5e77766243bb58d47122be18d7b8e3347bbf5/subgraph/README.md#fluence-subgraph-versioning

ref to https://www.notion.so/fluencenetwork/Subgraph-versioning-f88d7647c3954b558b439a9446f86f33#d781e7ad8ca042e292c4a9939f5faa32

### CODE REVIEW

File: subgraph/README.md [hunk 1]
```diff
@@ -123,17 +130,18 @@ npm run compile && npm run create:local && npm run deploy:local
...
 2. It deploys actual code to index contracts according to [configs](configs) and uses [subgraph.yaml](subgraph.yaml) as template (uses addresses and block number from config).
-3. WARN! Currently, after deploy of the new subgraph for the git commit you ought to update url in the [ts-client/src/indexerClient/config.ts](ts-client/src/indexerClient/config.ts) to sync ts-client and newly deployed subraph (ts-client has test for that check).
-4. When need to delete you need to delete subgraph you process manually command: `graph remove --node ${GRAPHNODE_URL} ${SUBGRAPH_NAME}` and delete from file currently. Thus, we free resources for current stand indexer.
+3. WARN! Currently, after deploy of the new subgraph for the git commit you ought to update url in the [ts-client/src/indexerClient/config.ts](ts-client/src/indexerClient/config.ts) to sync ts-client and newly deployed subgraph (**ts-client has test for that mapping**).
+4. When need to **delete deprecated subgraph** you manually process the following command: `npm run delete:{stage,kras,dar,local}`.
```
Comment (justprosh) [inline subgraph/README.md:143]: What does `deprecated` mean? Which versions will be deleted?

File: subgraph/fluence-graph.sh [hunk 2]
```diff
@@ -29,8 +29,14 @@ HELP
...
   case "$1" in
-    deploy|create|remove)
+    deploy|create)
       action="$1"
+      echo "1: $1"
+      shift
+      ;;
+    delete)
+      action="$1"
+      echo "2: $2"
```
Comment (justprosh) [inline subgraph/fluence-graph.sh:39]: is it for debug?

Comment (AlcibiadesCleinias) [reply subgraph/fluence-graph.sh]: fixed, it

File: subgraph/README.md [hunk 3]
```diff
@@ -186,5 +194,5 @@ graph deploy --product hosted-service --network mumbai <githubName>/fluence-deal
 - [ ] subgraph tests
 - [ ] fix docker versions
 - [ ] it is mb better to flag CU and etc as removed instead of deleting as it is in, e.g. `store.remove('ComputeUnit', computeUnitEntity.id)`
-- [ ] rm warn about to hexString instead of hex()
-- [ ] where is delete events? (e.g. peer deleted)
+- [x] rm warn about to hexString instead of hex()
```
Comment (AlcibiadesCleinias) [inline subgraph/README.md:197]: sorry to out into this PR

File: Makefile [hunk 4]
```diff
@@ -55,6 +55,10 @@ deploy-subgraph-%: ## Deploy subgraph to network {local, kras, dar, stage}
 	@make verify-command program=npm
 	@cd subgraph && npm run create:$* && npm run deploy:$*
 
+delete-subgraph-%: ## Delete subgraph from network {local, kras, dar, stage}, dnf to set SUBGRAPH_NAME_TO_DELETE
```
Comment (justprosh) [inline Makefile:58]: What si `dnf`? Where from I can get those names?

Comment (justprosh) [reply Makefile:58]: i don't see any list commands or are they stored somewhere?

Comment (AlcibiadesCleinias) [reply Makefile:58]: do not forget, will put full text

Comment (AlcibiadesCleinias) [reply Makefile:58]: > i don't see any list commands or are they stored somewhere?
`make help`

**Review by justprosh — APPROVED** [review]

**Pull request merged**